### DISCUSSION TITLE
TJC Display Wierd Display

### DISCUSSION THREAD

**Discussion by Dakweed666** [mriscoc/Ender3V2S1#1051] [General]
Latest version of mriscoc on a 3 V2 NEO, used the build from the correct page and also did the display with the update, still get glitches all the time, in this photo bottom right the z symbol is showing as an A, first time Im seeing this one, sometimes the fan icon above it is missing and also previews sometimes don't show, if it doesn't show, cancel it, load it again and then it may show. Any ideas?
![20230826_203001](https://github.com/mriscoc/Ender3V2S1/assets/141825054/331cd1bd-a5be-4294-b064-8586041c0334)

**Comment by mriscoc** [comment]
If you have your printer connected via USB, be sure that +VUSB is disabled. Screen glitches are caused by interferences on the communication link with the display unit, and can be cleared by going to the tune menu and selecting screen off/on.
